packages/amplify-appsync-simulator/src/velocity/value-mapper/array.ts

```diff
@@ -1,3 +1,4 @@
+import { JavaString } from './string';
```

Comment:
I don't think importing `JavaString` is the correct way to go here. The `mapper` argument passed to the constructor can do this for you.

Reply:
hey, 

Thank you for taking the time to review this. 

Will take a look at how this can be done using the mapper, and will also add tests

Reply:
Hi @cjihrig, 

I've made the changes, please do let me know if there is anything else that would be required. 

Thank you for your time.